featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -83,7 +122,18 @@ def generate_name(self, base_feature_names, child_entity_id,
 
 
 class Mode(AggregationPrimitive):
-    """Finds the most common element in a categorical feature."""
+    """Finds the mode of a feature.
+
+    Description:
+        Given a list of values, return the mode (or most
+        commonly repeated value in the list). If list is
+        empty, return `NaN`.
+
+    Examples:
+        >>> mode = Mode()
+        >>> mode([1, 2, 3, 4, 4, 5, None])
```

Comment:
let's make this example strings since the input type is discrete

Reply:
Fixed to strings (using colors)